lemur/certificates/service.py

```diff
@@ -117,6 +117,41 @@ def get_all_pending_cleaning(source):
     )
 
 
+def get_all_pending_cleaning_about_to_expire_certs(source, days_to_expire):
+    """
+    Retrieves all certificates that are available for cleaning: not attached to endpoint,
+    and within X days from expiration.
+
+    :param days_to_expire:
+    :param source:
+    :return:
```

Comment:
Please update the documentation for each param and return for clarity.

Reply:
thanks for the feedback